docs/user_guide/concepts/main_concepts/data_management.rst

```diff
@@ -170,6 +170,13 @@ But for Metadata, the data should be accessible to Flyte control plane.
 
 Data persistence is also pluggable. By default, it supports all major blob stores and uses an interface defined in Flytestdlib.
 
+Deleting Raw Data in Your Own Datastores
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Flyte does not offer a direct function to delete raw data stored in external datastores like ``S3`` or ``GCS``. However, you can manage deletion by configuring a lifecycle policy within your datastore service.
+
+If caching is enabled in your Flyte ``task``, ensure that the ``max-cache-age`` is set to be shorter than the lifecycle policy in your datastore to prevent potential data inconsistency issues.
+
 Practical Example
```

Comment:
Not sure how this example connects with the content immediately above. This is an example of data movement, which is good, so maybe it would be better placed at the beginning of the page?

Reply:
Hi @davidmirror-ops , thanks for your reviewing. Please let me clarify, do you suggest to place the example section to the top of the Doc?

Reply:
Hi @davidmirror-ops , I think you are right. The example section in this PR is indeed more relevant to data movement. To reduce ambiguity, I’ve moved the example section directly below the data movement section and relocated the Own Datastores section to the bottom of the page. Let me know if you have any further suggestions, thanks! [doc-link](https://flyte--5922.org.readthedocs.build/en/5922/user_guide/concepts/main_concepts/data_management.html)